target/linux/ramips/mt76x8/base-files/etc/board.d/02_network

```diff
@@ -201,7 +201,8 @@ ramips_setup_macs()
 	unielec,u7628-01-16m|\
 	wavlink,wl-wn575a3|\
 	wiznet,wizfi630s)
-		wan_mac=$(macaddr_add "$(mtd_get_mac_binary factory 0x28)" 1)
+		wan_mac=$(mtd_get_mac_binary factory 0x28)
```

Comment:
So, 0x28 is *:f1? Or is it *:f2?

Reply:
The OEM firmware is not consistant during initalisation (lan / wan mac change). Maybe it's related to the fact that in the oem firmware the wan adress is derived as eth0 mac + 1 which is a "don't" according to the link you sent.
Here you can see three screenshots.
There are three columns: This Patch / OEM fully booted / OEM during init
![image](https://user-images.githubusercontent.com/2125300/89473369-f4bd7a00-d782-11ea-84a4-61b573dcbf41.png)
Anyway with this LAN / WAN have the same mac adresses.